nemo_curator/modules/distributed_data_classifier.py

```diff
@@ -96,6 +150,126 @@ def _filter_documents(
         raise TypeError("filter_by must be a string or list type")
 
 
+def _run_classifier_helper(
+    df: "dask_cudf.DataFrame",
+    model: "HFModel",
+    labels: list[str],
+    max_chars: int,
+    batch_size: int,
+    label_col: str,
+    prob_col: str = None,
```

Comment:
Instead of having two variables `prob_col` and `keep_prob`, could you have just one variable `prob_col` and just set `keep_prob = prob_col is not None`?

Reply:
Resolved